guava/src/com/google/common/base/Ascii.java

```diff
@@ -629,4 +630,78 @@ private static int getAlphaIndex(char c) {
     // Fold upper-case ASCII to lower-case and make zero-indexed and unsigned (by casting to char).
     return (char) ((c | 0x20) - 'a');
   }
+
+  /**
+   * Returns the index within {@code sequence} of the first occurrence of the
+   * specified {@code subSequence}, ignoring the case of any ASCII alphabetic characters
+   * between {@code 'a'} and {@code 'z'} or {@code 'A'} and {@code 'Z'} inclusive, or
+   * {@code -1}, if no occurence is found.
+   *
+   * @since 16.0 // TODO: which number?
```

Comment:
You can change these to `@since NEXT`

Reply:
Thanks for letting me know! I will update them.